model/Core/Classes/ProfileIdentifier.md

```diff
@@ -4,11 +4,11 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Provides the model profile that the Element is specified in.
```

Comment:
suggest removing `model` and just use the word profile

Reply:
done